duke-core/src/main/java/no/priv/garshol/duke/cleaners/StopwordsCleaner.java

```diff
@@ -0,0 +1,72 @@
+
+package no.priv.garshol.duke.cleaners;
+
+import no.priv.garshol.duke.Cleaner;
+
+import java.io.*;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * A cleaner which removes english stopwords from a string.
+ */
+
+public class StopwordsCleaner implements Cleaner {
+    private LowerCaseNormalizeCleaner sub;
+    private String[] stopwords;
```

Comment:
Searching this array for stop words is very inefficient. Please use a HashSet instead. That will be much faster and also easier to use.